components/camel-as2/camel-as2-api/pom.xml

```diff
@@ -96,6 +96,13 @@
             <artifactId>camel-test-junit5</artifactId>
             <scope>test</scope>
         </dependency>
+        <!-- added for binary content transfer encoding test because Camel AS2 client doesn't support binary currently -->
+        <dependency>
+            <groupId>com.helger.as2</groupId>
+            <artifactId>as2-lib</artifactId>
+            <version>4.11.0</version>
+            <scope>test</scope>
```

Comment:
This should be in a property placeholder in parent pom. Don't use a static version inside the component pom

Reply:
Thanks! There were a lot to do and I forgot to do so while I was going to. Nice catch :+1: fixed